src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+                <h2 id="role-of-the-organizer-and-the-delegate">Role of the organizer and the Delegate</h2>
+                <p>The organizer and Delegate have several roles, but these can sometimes vary by region throughout the world. Here we want to describe exactly what the Delegate will do, and what the organizer (you) will need to do.</p>
+                <h3 id="choosing-the-date">Choosing the date</h3>
+                <p>Competitions have to be about 3 weeks apart if they’re within 100km of each other by WCA Regulations. Once you start with the organizing process, the Delegate will share a spreadsheet with you with upcoming competitions so we can ensure your competition doesn’t conflict with other competitions. Of course, the date must also work for the Delegate, organizer(s), and the venue itself. It’s best to choose a date that doesn’t conflict with a major holiday (like winter break) or other dates when students might be busy or away.</p>
+                <h3 id="the-venue">The venue</h3>
+                <p>The venue is a key part of the competition experience. The organizer is entirely responsible for choosing and booking the venue. Of course, the Delegate needs to approve the venue (do NOT book without explicit Delegate approval). In general, you should look for a venue that has</p>
+                <ul>
+                        <li>enough space for everyone (there is usually extra person per competitor, so the room should be able to easily hold double the competitor limit)</li>
+                        <li>an internet connection, preferably WiFi</li>
+                        <li>a reasonable temperature (not too hot or too cold)</li>
+                        <li>a reasonable price (this will be discussed later)</li>
+                        <li>a PA system (nice to have, but not a requirement)</li>
+                        <li>a projector (even less of a requirement but still nice to have)</li>
+                        <li>tables (rectangular competition tables are a must (or you’ll need to rent some), and other tables for the audience and competitors to sit at are nice)</li>
+                        <li>lighting that is appropriate for solving that will be relatively consistent throughout the day</li>
+
+                </ul>
+                <p>When you start writing up a budget, you should have a good idea of how much the venue will cost. Schools and universities can often provide low-cost venues, but that’s not always an option. Be sure to make note of required custodians or police officers, as this usually costs extra. Places of worship like churches may be good candidates, but their space is often either too small or not suitable (ie full of pews). Community centers and similar places are also good venues to try.</p>
```

Comment:
This paragraph starts with budget and ends with potential venue suggestions.  Keep on topic, move budget comments to the budget section.